Model/Adminhtml/Checkout.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+namespace Affirm\Telesales\Model\Adminhtml;
+
+use Astound\Affirm\Gateway\Helper\Util;
+use Astound\Affirm\Model\Config as ConfigAffirm;
+use Astound\Affirm\Model\Ui\ConfigProvider;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\App\ProductMetadataInterface;
+use Magento\Framework\Controller\Result\JsonFactory;
+use Magento\Framework\HTTP\ZendClientFactory;
+use Magento\Framework\Model\Context;
+use Magento\Framework\Module\ResourceInterface;
+use Magento\Framework\Registry;
+use Magento\Framework\Url as Url;
+use Magento\Sales\Api\Data\OrderStatusHistoryInterface;
+use Magento\Sales\Api\OrderRepositoryInterface;
+use Affirm\Telesales\Model\Config as ConfigAffirmTelesales;
+
+class Checkout extends \Magento\Framework\Model\AbstractModel
+{
+    /**#@+
+     * Define constants
+     */
+    const API_STORE_CHECKOUT_PATH = '/api/v2/checkout/store';
+    const API_STORE_RESEND_PATH = '/api/v2/checkout/resend';
+    const API_STORE_READ_PATH = '/api/v2/checkout/';
+    const API_CHARGES_PATH = '/api/v2/charges/';
+    const CHECKOUT_FLOW_TYPE = 'In-Store';
```

Comment:
Do we not have a flow type called telesales

Reply:
updating constant to `const CHECKOUT_MODE = 'telesales';`